openedx_events/enterprise/data.py

```diff
@@ -0,0 +1,53 @@
+"""
+Data attributes for events within the architecture subdomain `enterprise`.
+
+These attributes follow the form of attr objects specified in OEP-49 data
+pattern.
+"""
+import attr
+
+
+@attr.s(frozen=True)
+class TrackingEvent:
```

Comment:
This should be a much more specific name, right?

Reply:
How about LicenseLifeCycle? Based on: https://github.com/openedx/license-manager/blob/d298a722d292ba91d4b1c8afa88c4bd927752492/docs/segment_events.rst#event-nested-data-field-specification